ext/include/opentelemetry/ext/zpages/latency_boundaries.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+// include libraries
+#include <chrono>
+#include <climits>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace ext
+{
+namespace zpages
+{
+
+/**
+ * Latency_Boundary_Name enum is used to index into the Latency_Boundaries vector that is declared later on,
+ * using this enum lets you access the Latecy_Boundary at each index without using magic numbers
+ */
+typedef enum LatencyBoundaryName
+{
+  k0MicroTo10Micro = 0,
+  k10MicroTo100Micro,
+  k100MicroTo1Milli,
+  k1MilliTo10Milli,
+  k10MilliTo100Milli,
+  k100MilliTo1Second,
```

Comment:
In C++, you can just do
```
enum LatencyBoundaryName
{
  ...,
  ...
};
```

Reply:
Yeah it worked. Got rid of the typedef thanks!